### ISSUE TITLE
Moodle 3.7/3.8, Classic/more -theme and H5P, embedded h5p acitivities not visible

### ISSUE THREAD

**Issue by helivee** [h5p/h5p-moodle-plugin#329]
We are using in several Moodle-environments h5p and h5p-version 1.15 works fine with embedded activities. 

Now we have tested moodle versions 3.7 and 3.8 with h5p 1.18 and h5p 1.20.1 and have the same problem with all cases. Embedded activities not shown right, the frame is there but only a really thin box at the left side of the screen. After many hours work I decided to change the theme to Boost and problem was solved. Everything shows normally. 

How to solve this bug/problem with classic theme too? Is this a bug or what? We have over hundred customers who are using classic theme, so this might be a big problem in the future.

**Comment by helivee** [comment]
I tested combination moodle 3.8 and h5p 1.15 and all worked fine with classic theme too. So the problem comes out with later versions of h5p.

**Comment by arherstal** [comment]
Since the last update of H5P Plugin
![moodle-h5p-01](https://user-images.githubusercontent.com/62618714/77465013-539fd980-6e08-11ea-8328-948744bc2db9.png)
![moodle-h5p-02](https://user-images.githubusercontent.com/62618714/77465015-54387000-6e08-11ea-93d1-a3549231801d.png)

 the h5p hub doesn't display correctly when adding "new activities", the width of the h5p hub iframe is stucked at 300px.
I'm using default template on moodle.

Tested on firefox & chrome.

What am i missing ?

H5P version 1.20.2
Moodle version 3.8.2+

Screenshot in attachement.

**Comment by thomasmars** [comment]
Seems this commit in Moodle https://github.com/moodle/moodle/commit/eebe0031041b2314e5bc7bae7f770049d48b6083 changes how the 'html' mod_form element behaves. I'll ask Moodle if this is intended, and how we can resolve it.

**Comment by arherstal** [comment]
I fixed the problem temporary by modifying .mform fieldset style in the forms.scss file of the boost theme. That's the style of the fieldset containing the H5P Hub iframe.

```
.mform fieldset {
    margin-bottom: $spacer / 2;
    border-bottom: $border-width solid $table-border-color;
    **width : 100%;**
}
```

**Comment by stronk7** [comment]
Just cross-linking to https://tracker.moodle.org/browse/MDL-68269 , where the problem will be fixed, (hopefully) soon.

If you have any information or detail worth being shared, please add it there.

Ciao :-)

**Comment by thomasmars** [comment]
The issue with the editor should be fixed in the latest version of Moodle: 3.8.2+ (Build: 20200408)
